braveautogen-chromium-upgrade.py

```diff
@@ -9,7 +9,7 @@
 secret_file = open('github.secret', 'r')
 token_string = secret_file.readline().rstrip("\n\r")
 
-wiki_file = open('wikitemplate-cr-upgrade.md', 'r')
+wiki_file = open('wikitemplate.md', 'r')
```

Comment:
we'll probably want to remove `braveautogen-chromium-upgrade.py` as it's basically going to be same thing as `braveautogen.py`.